x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/ErrorsOverviewLink.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import euiThemeLight from '@elastic/eui/dist/eui_theme_light.json';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import styled from 'styled-components';
+import { idx } from 'x-pack/plugins/apm/common/idx';
+import { KibanaLink } from 'x-pack/plugins/apm/public/components/shared/Links/KibanaLink';
+import { legacyEncodeURIComponent } from 'x-pack/plugins/apm/public/components/shared/Links/url_helpers';
+import { Transaction } from '../../../../../typings/es_schemas/Transaction';
+import { fontSize } from '../../../../style/variables';
+
+const LinkLabel = styled.span`
+  font-size: ${fontSize};
+  color: ${euiThemeLight.euiColorDanger};
+`;
+
+interface ErrorsOverviewLinkProps {
+  errorCount: number;
+  transaction: Transaction;
+  showLabel?: boolean;
+}
+
+export const ErrorsOverviewLink: React.SFC<ErrorsOverviewLinkProps> = ({
```

Comment:
nit: what do you think about `RelatedErrorsBadge` or `ErrorCountBadge` or something else that more describes what the component is, and not where it links to.